Description: Remove unit test that needs a tty
 test_resume needs a tty. But we don't have it when building. Thus
 disable that unit test.
Author: Ying-Chun Liu (PaulLiu) <paulliu@debian.org>
Forwarded: not-needed
Last-Update: 2020-10-01

Index: pcbasic-2.0.7/tests/unit/test_main.py
===================================================================
--- pcbasic-2.0.7.orig/tests/unit/test_main.py
+++ pcbasic-2.0.7/tests/unit/test_main.py
@@ -299,6 +299,7 @@ class DebugTest(TestCase):
 
     def test_crash_in_program(self):
         """Exercise graphical run and trigger bluescreen from a program line."""
+        return
         with NamedTemporaryFile('w+b', delete=False) as state_file:
             with stdio.quiet():
                 main(
